### PR TITLE
General formatting fixes and added some TODOs

### LINKED ISSUE #10: Refactor General Architecture
Right now, our code architecture is pretty messy - all in one file, everything pushed together. We should take a look at how to maybe split it into several files, to make it easier to review and work on specific parts of our code

### PR DESCRIPTION
**Pull request by TrueMichato**

### CODE REVIEW

File: whatsapp.py [hunk 1]
```diff
@@ -1,185 +1,27 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
+from parsing_tools import *
```
Comment (mayamz) [inline whatsapp.py]: maybe I'm wrong, but isn't the convention is to first import global libraries, and then import localised?

File: whatsapp.py [hunk 2]
```diff
@@ -189,54 +31,55 @@ def main(path,name):
     df.head()
 
     # drop "Media omitted"
-    media_df = df[df["text"]=="<Media omitted>"]
-    df = df[df["text"]!="<Media omitted>"]
```
Comment (mayamz) [inline whatsapp.py:193]: Now I get why I was instructed to move files in a different PR, I can see the old commit here as files changes, but not the ones in the files that were moved. Writing this just so we'll remember for next time

File: defaults.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import datetime
+
```
Comment (mayamz) [inline defaults.py:2]: liked the file. great file.
(not sarcasm)

File: parsing_tools.py [hunk 4]
```diff
@@ -0,0 +1,106 @@
+import datetime
+import re
+from defaults import TIMEZONE_SHIFT, DEFAULT_FILE_TYPE, FILE_TYPES
```
Comment (mayamz) [inline parsing_tools.py:3]: I think we should import * from default. That way if we add more defaults the list won't be as long

File: whatsapp.py [hunk 5]
```diff
@@ -289,4 +133,3 @@ def main(path,name):
 if __name__ == "__main__":
     path = "/Users/mayam/Downloads/chats/WhatsApp Chat with "
```
Comment (mayamz) [inline whatsapp.py:134]: Do you change the path and name each time you run? seems uncomfortable. Would you prefer we'll find another way?

File: whatsapp.py (hunk 1 above)
Comment (TrueMichato) [reply whatsapp.py]: Correct, I was unaware of this convention. Fixed

File: parsing_tools.py (hunk 4 above)
Comment (TrueMichato) [reply parsing_tools.py:3]: True, but if we add, for example, a 100,000,000  items sized dict to our defaults that comes into play only in statistics....

File: whatsapp.py (hunk 5 above)
Comment (TrueMichato) [reply whatsapp.py:134]: Yeah, maybe we can just have a specific place in our project for placing chats in, would make much more sense

Comment (mayamz) [reply whatsapp.py:134]: let's add a todo, it's not for this PR

File: parsing_tools.py (hunk 4 above)
Comment (mayamz) [reply parsing_tools.py:3]: We've decided we'll leave it like that for now

**Review by mayamz — APPROVED** [review]

**Pull request merged**